onnxruntime/contrib_ops/cpu/nchwc_ops.cc

```diff
@@ -7,90 +7,128 @@
 namespace onnxruntime {
 namespace contrib {
 
-#define ONNX_CPU_OPERATOR_TYPED_NCHWC_KERNEL(name, ver, type, builder, ...) \
-  ONNX_OPERATOR_TYPED_KERNEL_EX(name, kMSNchwcDomain, ver, type, kCpuExecutionProvider, builder, __VA_ARGS__)
+Status ReorderInput::Compute(OpKernelContext* context) const {
+  const auto* X = context->Input<Tensor>(0);
+  const auto& X_shape = X->Shape().GetDims();
+  const auto X_rank = X_shape.size();
+  ORT_ENFORCE(X_rank >= 2);
```

Comment:
>2 [](http://example.com/codeflow?start=24&length=1)

do you mean 3?

Reply:
It was supposed to be any image tensor that has N+C, but I don't have tests to verify that, so I dropped it back to what it was before, checking that X_rank must be 4.